ivadomed/training.py

```diff
@@ -54,6 +55,8 @@ def train(model_params, dataset_train, dataset_val, training_params, path_output
     # Write the metrics, images, etc to TensorBoard format
     writer = SummaryWriter(log_dir=path_output)
 
+    set_seed(random_seed)
```

Comment:
Coming back to a few points I mentioned earlier, **when** to set the seed is actually a great and not so simple question. If we do centralize the seed somehow, we probably want to set it sometimes as early as possible. However, current seed is in the `config` file so maybe a better place to set seed is in the loading function for config? What are your thoughts on this? Ideally, we probably only need to set the seed once as well.

Reply:
The way the ivadomed codebase is, I couldn't come up with a better fix (I have to analyse more, and we need to discuss), than to have one in the `training` and another in the `loader`. 
Usually, the seed is to be set once. Anyways, I don't think the current implementation hurts anything, except the issue with where the key needs to be in the config file.